tests/kernel/install_ltp.pm

```diff
@@ -224,12 +224,24 @@ sub add_ltp_repo {
     zypper_ar($repo, name => 'ltp_repo');
 }
 
-sub install_from_repo {
-    my $pkg = get_var('LTP_PKG', (want_stable && is_sle) ? 'qa_test_ltp' : 'ltp');
+sub get_default_pkg {
+    return (want_stable && is_sle) ? 'qa_test_ltp' : 'ltp';
+}
 
-    zypper_call("in --recommends $pkg");
-    script_run "rpm -qi $pkg | tee /opt/ltp_version";
-    assert_script_run "find " . get_ltproot() . q(/testcases/bin/openposix/conformance/interfaces/ -name '*.run-test' > ~/openposix-test-list);
+sub install_from_repo {
+    my $default_pkg      = get_default_pkg;
+    my @pkgs             = (get_var('LTP_PKG', $default_pkg));
+    my $install_both_pkg = $pkgs[0] eq 'both';
+    my $want_32bit       = undef;
+
+    @pkgs = ($default_pkg, $default_pkg . '-32bit') if ($install_both_pkg);
+
+    for my $pkg (@pkgs) {
+        $want_32bit = $pkg =~ m/32bit/ if ($install_both_pkg);
+        zypper_call("in --recommends $pkg");
+        script_run "rpm -qi $pkg | tee /opt/ltp_version";
```

Comment:
I like to see this in the tests... 
```suggestion
        record_info('LTP_VERSION', script_output("rpm -qi $pkg | tee /opt/ltp_version"));
```

Reply:
That'd be certainly useful, feel free to implement it.

Reply:
BTW this should be not just in `install_ltp.pm`, but also in `schedule_tests()`, which prints it during test run.

Reply:
I think you already did, nice!